pex/bin/pex.py

```diff
@@ -637,7 +637,11 @@ def configure_clp():
         "sources, requirements, their dependencies and other options."
```

Comment:
Right here is probably the best place to mention the @ handling. If you want to add in some newlines so the mention gets its own paragraph - that works for me.